NOTICE

```diff
@@ -0,0 +1,5 @@
+Jenkins Remoting
+Copyright (c) 2004-2016, Sun Microsystems, Inc., Kohsuke Kawaguchi, Daniel Dyer, Stephen Connolly, and others
+
+This product includes software developed at
```

Comment:
Maybe you want to put the detailed NOTICE to the root as well. Otherwise it's strange to have detailed licenses in secondary assemblies only